config.go

```diff
@@ -40,6 +40,7 @@ func newDefaultConfig() *Config {
 		ServerReadTimeout:           5 * time.Second,
 		ServerWriteTimeout:          10 * time.Second,
 		SkipOpenIDProviderTLSVerify: false,
+		OpenIDProviderProxy:         "",
```

Comment:
I think we can drop that one .. it's gonna default to "" anyhow; though if you wanna keep it for informational purposes can we order them alphabetically (i'm slight ocd on that :-))

Reply:
Done, removed.